app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryViewCallback.kt

```diff
@@ -15,30 +15,21 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.zim_manager.library_view;
+package org.kiwix.kiwixmobile.zim_manager.library_view
 
-import java.util.LinkedList;
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book;
+import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book
+import java.util.*
```

Comment:
Can we add this `java.util.LinkedList` instead of this?

Reply:
I'll close this PR and open a new one by incorporating all the changes. Thank you!